.eslintignore

```diff
@@ -1 +1,2 @@
 modules/default/calendar/vendor/*
+tests/configs/port_variable.js
```

Comment:
why are these two ignores necessary? locally nothing complains about them....

Reply:
you are right, will remove them. Had first `port_variable.js` (forgot `.template` at the end) and with `.js` the tests failed.